### PR TITLE
Works section

### PR DESCRIPTION
**Pull request by Daniel-Kigozi**
-Add about my works page

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -85,4 +87,185 @@
...
+    border-radius: 5px;
+    width: 90%;
+    margin-top: 10px;
+  }
+
+  .first-item {
+    border-radius: 8px 8px 5px 5px;
+    margin-top: 200px;
+    background-color: white;
+    width: 100%;
```
Comment (Ugboaja-Uchechi) [inline styles.css]: - [ ] Kindly make sure you use `CSS Grid` in the main container that contains all of the cards in the `work` section as stated in the project requirements 🙏

File: styles.css [hunk 2]
```diff
@@ -85,4 +87,185 @@
   .social-icons li {
     margin: 0 10px;
   }
+
+  .list-item {
+    background-color: #ebf0ee;
+    border: 1px solid #d0d9d4;
+    border-radius: 5px;
+    width: 90%;
+    margin-top: 10px;
```
Comment (Ugboaja-Uchechi) [inline styles.css]: - [ ] Kindly use `flexbox` to lay out all individual cards of the `work` section as stated in the project requirement 🙏 .

**Review by Ugboaja-Uchechi — CHANGES_REQUESTED** [review]
Hi @Daniel-Kigozi,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights

- Linters are passing ✔️ 
- README is descriptive 👍 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Daniel-Kigozi** [comment]
@Ugboaja-Uchechi I have corrected the changes for your review

**Review by sumairq — APPROVED** [review]
### Hi @Daniel-Kigozi  , 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Github flow
✔️Descriptive PR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**